### ISSUE TITLE
how to read whatsapp messages ?

### ISSUE THREAD

**Issue by jishnurajm** [adonespitogo/AdoBot#24]
is there any way to read whats app messages ?

**Comment by KARANS0001** [comment]
Sorry I don't know

**Comment by U65535F** [comment]
Yes possible. We need to change the WhatsApp version to the legacy WhatsApp by reinstalling it, we can access keys and message databases easily without root if we have the legacy version of WhatsApp.
